src/main/java/seedu/edulink/logic/parser/AddressBookParser.java

```diff
@@ -85,6 +87,12 @@ public Command parseCommand(String userInput) throws ParseException {
             return new UndoCommand();
         case TagCommand.COMMAND_WORD:
             return new TagCommandParser().parse(arguments);
+
+        case DeleteTagCommand.COMMAND_WORD:
+            return new DeleteTagCommandParser().parse(arguments);
```

Comment:
edit the AddressBookParserTest file to test these lines of code as well

Reply:
Thank you for your comments. Issue resolved.